content/webapp/components/GuideTextItem/GuideTextItem.tsx

```diff
@@ -33,12 +34,15 @@ const GuideTextItem: FunctionComponent<{
           )}
         </Tombstone>
 
-        {caption && (
-          <CaptionWrapper>
-            <Caption>
-              <PrismicHtmlBlock html={caption} />
-            </Caption>
-          </CaptionWrapper>
+        {(caption || additionalNotes) && (
+          <Wrapper>
+            {additionalNotes && <PrismicHtmlBlock html={additionalNotes} />}
+            {caption && (
+              <Caption>
+                <PrismicHtmlBlock html={caption} />
+              </Caption>
+            )}
+          </Wrapper>
```

Comment:
not important but IIUC, both can appear, so could they just be split within their own condition?

Reply:
I want them to be in the same Wrapper, but only want the Wrapper to render if one or both exist